src/shared/components/NoOqlWarning.tsx

```diff
@@ -1,16 +1,23 @@
 import * as React from "react";
+import {ResultsViewPageStore} from "../../pages/resultsView/ResultsViewPageStore";
+import {observer} from "mobx-react";
 
-//<div className="alert alert-info" role="alert">
-
-const NoOqlWarning = (
-    <span style={{color:"#74bedb", fontFamily:"arial", fontSize:"13px", fontWeight:"bold"}}>
-        <span style={{marginRight:4, verticalAlign:"middle"}}>
-            <i
-                className="glyphicon glyphicon-info-sign"
-            />
-        </span>
-        This tab does not reflect the OQL specification from your query.
-    </span>
-);
-
-export default NoOqlWarning;
\ No newline at end of file
+@observer
+export default class NoOqlWarning extends React.Component<{store:ResultsViewPageStore}, {}> {
+    render() {
+        if (this.props.store.queryContainsOql) {
```

Comment:
would rather see this outside in parent view, but no biggie

Reply:
just wanted to simplify use since it would always be the same